src/convnets/convmixer.jl

```diff
@@ -0,0 +1,68 @@
+"""
+    convmixer(planes, depth; inchannels = 3, kernel_size = (9, 9), patch_size = 7,
+              activation = gelu, nclasses = 1000)
+
+Creates a ConvMixer model.
+([reference](https://arxiv.org/abs/2201.09792))
+
+# Arguments
+- `planes`: number of planes in the output of each block
+- `depth`: number of layers
+- `inchannels`: number of channels in the input
+- `kernel_size`: kernel size of the convolutional layers
+- `patch_size`: size of the patches
+- `activation`: activation function used after the convolutional layers
+- `nclasses`: number of classes in the output
+"""
+function convmixer(planes, depth; inchannels = 3, kernel_size = (9, 9), patch_size = 7,
+                   activation = gelu, nclasses = 1000)
+  stem = conv_bn((patch_size, patch_size), inchannels, planes, activation; preact = true, stride = patch_size)
+  blocks = [Chain(SkipConnection(Chain(conv_bn(kernel_size, planes, planes, activation;
+                                               preact = true, groups = planes, pad = SamePad())...), +),
+                  conv_bn((1, 1), planes, planes, activation; preact = true)...) for _ in 1:depth]
+  pool = Chain(AdaptiveMeanPool((1, 1)), MLUtils.flatten)
+  head = Dense(planes, nclasses)
+  return Chain(Chain(stem..., blocks..., pool),  head)
+end
+
+convmixer_config = Dict(:base => Dict(:planes => 1536, :depth => 20, :kernel_size => (9, 9),
+                                         :patch_size => 7),
+                        :small => Dict(:planes => 768, :depth => 32, :kernel_size => (7, 7),
+                                        :patch_size => 7),
+                        :large => Dict(:planes => 1024, :depth => 20, :kernel_size => (9, 9),
+                                         :patch_size => 7))
+
+"""
+    ConvMixer(mode::Symbol = :base; inchannels = 3, activation = gelu, nclasses = 1000)
+
+Creates a ConvMixer model.
+([reference](https://arxiv.org/abs/2201.09792))
+
+# Arguments
+- `planes`: number of planes in the output of each block
+- `depth`: number of layers
+- `inchannels`: number of channels in the input
+- `kernel_size`: kernel size of the convolutional layers
+- `patch_size`: size of the patches
+- `activation`: activation function used after the convolutional layers
+- `nclasses`: number of classes in the output
```

Comment:
these do not correspond to the signature

Reply:
Fixed now!